app/src/main/java/com/example/campusguide/map/GoogleMapAdapter.kt

```diff
@@ -61,6 +61,10 @@ class GoogleMapAdapter : Map {
         adapted.setOnInfoWindowClickListener(infoWindowClickListener)
     }
 
+    override fun setOnInfoWindowCloseListener(infoWindowCloseListener: GoogleMap.OnInfoWindowCloseListener) {
```

Comment:
Nice addition to our listener patter! will this get called when any infoWindow gets closed?

Reply:
This will get called when a building info-window gets closed! I decided to not set a listener for the info-window that pop-ups from selecting one of the search results since a user might want to close the info-window, look around the area, and tap on the marker again.